cli/zone.go

```diff
@@ -93,7 +92,7 @@ func runGetZone(cmd *cobra.Command, args []string) {
 		log.Errorf("Object %d: no zone config found", id)
 		return
 	}
-	fmt.Fprintln(osStdout, rows[0][1])
+	fmt.Println(rows[0][1])
```

Comment:
hadn't checked the actual code, now that I did no idea. Also wonder why the above is needed.